helm/istio/charts/kiali/templates/kiali-cr.yaml

```diff
@@ -8,29 +8,47 @@ spec:
   istio_namespace: "{{ .Release.Namespace }}"
 
   auth:
+{{- if .Values.dashboard }}
```

Comment:
Does helm let us do:
{{- if and (.Values.dashboard (not (and (.Values.dashboard.user) (.Values.dashboard.passphrase)))) }}

In other words, do we need else strategy: login else strategy: login?